doc/sphinx-guides/source/admin/big-data-administration.rst

```diff
@@ -0,0 +1,312 @@
+remote server which the remote server can then, if it is capable of validating them, use to reject requests not approved by Dataverse. In this configuration,
+users can restrict and embargo files and Dataverse and the remote server will cooperate to manage access control. Another alternative, with a more advanced
+remote store, would be, instead of using URLs that directly enable download of the file, to use URLs that point to a landing page at the remote server that 
+may require the user to login, or go through some other authentication/validation process before being able to access the file.
+
+Dataverse considers remote storage to be read-only, or, in cases where the remote service does not provide a way for Dataverse to download the file bytes
+(due to access control or because the URL refers to a landing page), inaccessible. Depending on whether Dataverse can access the bytes of the file,
+functionality such as ingest and integrity checking may or may not be possible. If the file bytes are not accessible the remote store in Dataverse should be
+configured to disable operations that attempt to access the file (see the files-not-accessible-by-dataverse in :ref:`trusted-remote-storage`).
+Regardless of whether the remote files can be read, local storage of other datafiles and auxiliary files in the same dataset is possible. Support for such files is handled by configuring a 'base' store with the remote store that is used for these purposes. (This means that while
+files added as remote remain on the remote store, other files in the dataset, and potentially thumbnails and the ingested TSV format of remote files would be managed by Dataverse
+in the base store. If ingest is not desired, the ingest size limit for the store can be set to 0 bytes).
+
+
+Benefits: 
+
+- This is a relatively simple way to off-load the management of large and/or sensitive data files to other organizations while still providing Dataverse's overall capabilities for dataset curation and publication to users.   
+- If the store has been configured with a remote-store-name or remote-store-url, the dataset file table will include this information for remote files. These provide a visual indicator that the files are not managed directly by Dataverse and are stored/managed by a remote trusted store.
+
+Challenges:
+
+- Currently, remote files can only be added via the API. (This may be addressed in future versions).
+- Remote files can only be added after the dataset is created in the UI (and therefore has an id and PID for use with the API). However, the UI will still allow upload of files to the base store (at dataset creation and when editing), which could be confusing.
+- As Dataverse is relying on the remote service to maintain the integrity and availability of the files, it is likely that the Dataverse site admin will want to have a formal agreement with the remote service 
+  operator about their policies.
```

Comment:
I would put this one first. Originally Dataverse was an archive. We resisted trusting some other system to house the data.